cmd/ebrelayer/txs/signature.go

```diff
@@ -8,19 +8,13 @@ import (
 
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/crypto"
-	"github.com/joho/godotenv"
```

Comment:
This could be used in place of the removed line and then its still possible to use `.env` for local development too:

`import _ "github.com/joho/godotenv/autoload"`

Reply:
Nice - will add.